src/main/java/org/lithium/Response.java

```diff
@@ -0,0 +1,49 @@
+package org.lithium;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import twitter4j.Status;
+
+import java.util.List;
+
+public class Response {
+    private String guid;
```

Comment:
Although not strict, it is better practice to assign a default value when declaring a non-final variable.